src/store/modules/wallet.js

```diff
@@ -65,6 +65,22 @@ export default {
         for (let addr in state.changeAddresses) {
           Vue.set(state.changeAddresses[addr], 'privKey', cashlib.PrivateKey.fromObject(state.changeAddresses[addr].privKey))
         }
+        for (const utxo of Object.values(state.utxos)) {
+          if (utxo.type === 'p2pkh') {
+            // Does not keep it's privKey with it because we like edge cases.
+            continue
+          }
+          Vue.set(utxo, 'address', cashlib.Address.fromObject(utxo.address.valueOf()))
```

Comment:
Vue.set is used to manually trigger reactivity when adding a new property to an object. I'm not sure it's necessary here because 'address' already exists.

Reply:
The value changes though.  I'm not sure though if anything is calculated from this.  It probably doesn't need reactivity at all?